src/dao/services/DAOService.ts

```diff
@@ -77,37 +77,11 @@ async function fetchMultiSigDAOs(eventTypes?: string[]): Promise<MultiSigDAODeta
 
   const multiSigEventResults = logs.map(async (log): Promise<MultiSigDAODetails> => {
     const argsWithName = getEventArgumentsByName<MultiSigDAOCreatedEventArgs>(log.args, ["owners", "webLinks"]);
-    const daoSolidityAddress = (await DexService.fetchContractId(argsWithName.daoAddress)).toSolidityAddress();
-    const safeDAOSolidityAddress = (await DexService.fetchContractId(argsWithName.safeAddress)).toSolidityAddress();
+    const { daoAddress: accountId } = argsWithName;
```

Comment:
Same here - update the variable names to accurately reflect the new event data. A `daoAddress` is not semantically the same as an `accountId`. This may require variable renaming in multiple files.

Reply:
accountId is the EVMAddress for MultiSig but for FT and NFT DAOs it is AccountId. so with this PR I have not changed the name of the variable. I wull update it with my next PR, in which I am making changes in the FT and NFT DAOs.